### DISCUSSION TITLE
about hard disk space

### DISCUSSION THREAD

**Discussion by xjctrl** [wix/Detox#3104] [General]
Hi, Before run test, my storage left 6.53G, after run 20 minutes, no fail case, so no images or videos were generated, it left 4.69G, I don't know if it is normal, if not , how can I solve this?

**Comment by noomorph** [comment]
Running AVD for the first time might use extra space. Try **WinDirStat** (**Disk Inventory X**, **KDirStat** - depending on your OS) to get some insights.

**Reply by xjctrl** [reply]
Thanks for your reply. I will use disk inventory x to get some insights. Add one thing: With the test execution, the disk space will be less and less. When all cases are completed and the disk space is restored to 6.53G.

**Reply by noomorph** [reply]
Well, sounds like a correct behaviour. Good luck with the investigation, and since it is a temporary disk usage, be creative: killall -9 in the middle, monitor temporary folders, etc. Can't give you a recipe, sorry. Come back if you find anything abnormal.

**Reply by xjctrl** [reply]
ok, Thanks!
